extension/js/common/api/shared/api-error.ts

```diff
@@ -27,6 +27,7 @@ interface RawAjaxErr {
 
 abstract class AuthErr extends Error {}
 export class GoogleAuthErr extends AuthErr {}
+export class EnterpriseServerAuthErr extends AuthErr {}
 export class BackendAuthErr extends AuthErr {}
```

Comment:
Are there any differences between `EnterpriseServerAuthErr` and `BackendAuthErr`? Can we merge them into single `EnterpriseServerAuthErr`?

Reply:
Yeah. I think we can